PictureViewMaster/PictureMasterView/PictureMasterImageView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  PictureMasterImageView.swift
+//  PictureViewMaster
+//
+//  Created by El gera de la gente on 4/6/16.
+//  Copyright © 2016 Inaka. All rights reserved.
+//
+
+import UIKit
+
+@objc protocol MasterViewDelegate: NSObjectProtocol {
```

Comment:
It should be renamed to `PictureMasterImageViewDelegate`. Right now it sounds confusing about where it comes from.